packages/frontend/src/components/Info/Accordion.tsx

```diff
@@ -24,33 +35,37 @@ export const InfoAccordion = () => {
           </StyledHeader>
           <StyledContent>
             <RuleText>
-              Total number of 100 raffle tickets is devided between auction pool and raffle pool. Auction winners will
-              win a Voucher Code that is redeemable for a Free Devcon ticket. Raffle winners will receive a voucher code
-              that is redeemable for 1 Devcon Ticket at Full Price.
+              Total number of {totalCount} raffle tickets is devided between auction pool and raffle pool. Auction
+              winners will win a voucher Code that is redeemable for a free Devcon ticket. Raffle winners will receive a
+              voucher code that is redeemable for 1 Devcon Ticket at full price.
             </RuleText>
             <Rule
-              heading="Auction pool: 20"
+              heading={`Auction pool: ${auctionWinnersCount}`}
               rule="Tickets from auction pool will be distributed to the highest bidding participants. The cost of a Devcon
               ticket in that pool is equal to the bid. All proceeds will go towards X Public Good."
```

Comment:
I'm guessing this is a placeholder and hasn't been decided yet?

Reply:
I assume this is a heading, but I could be wrong